src/ansys/grantami/recordlists/_models.py

```diff
@@ -978,7 +983,9 @@ def _to_model(self) -> models.GsaListAuditLogSearchRequest:
         logger.debug("Serializing AuditLogSearchCriterion to API model")
         model = models.GsaListAuditLogSearchRequest(
             list_actions_to_include=(
-                [item.value for item in self.filter_actions] if self.filter_actions else None
+                [models.GsaListAction[item.value] for item in self.filter_actions]
```

Comment:
This is a change to behavior. Instead of just using the string, we now set the value to the correct enum object.